content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -109,3 +109,26 @@ spec:
       weight: 10
 EOF
 {{< /text >}}
+
+### Authorization policy attachment
+
+In a scenario where policy requires application layer attributes, such as HTTP verbs, a waypoint proxy is required. It is important to understand that ztunnel cannot meaningfully enforce any policy that requires L7 parsing and so if they are present in your authorization policy which is enforces by ztunnel this will become a DENY rule as part of a safer "fail closed" security posture.
+
+This table is based on the following invariants:
+
+1. The source pod is a normal pod which has ztunnel enabled.
+1. Redirection to the waypoint is configured correctly.
+1. The waypoint is configured with the `istio.io/waypoint-for` label set to `service`.
+
+| Waypoint* | Attachment Style | Resources | Source Identity | Enforced By |
+| --- | --- | --- | --- | --- |
+| no | Selector | Pod | n/a | DENY destination ztunnel |
+| yes | Selector | Pod | n/a | DENY destination ztunnel |
+| no | _empty**_ | Namespace | n/a | DENY destination ztunnel |
+| yes | _empty**_ | Namespace | n/a | DENY destination ztunnel |
+| yes | `targetRefs` | Service | client pod | waypoint |
+| yes | `targetRefs` | Gateway | client pod | waypoint |
+
+* Whether or not there is already a waypoint is in the traffic path.
+
+** If no attachment is specified the policy will be treated as Namespace scoped.
```

Comment:
Namespace scoped but only for L4? This is same as line 127 & 128, correct?

Reply:
It was stated that having a blank cell in the table was confusing so I was trying to convey that if you don't have a selector or targetRef then you get namespace scope enforced at the ztunnel and this extra note was intended to clarify what _empty_ in the cell means. Maybe it's a bad idea or just not worded correctly

Reply:
I updated the wording of this line to hopefully clarify it a little. Ultimately maybe we should just change `_empty**_` though so that a footnote isn't required